net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild

```diff
@@ -109,13 +112,20 @@ src_install() {
 		doins id_resolver.conf
 	fi
 
-	systemd_dotmpfilesd "${FILESDIR}"/nfs-utils.conf
```

Comment:
The tmpfiles rules have been dropped, are the dirs now created by the service units or what?